iis-workflows/iis-workflows-ingest-pmc/src/test/java/eu/dnetlib/iis/workflows/ingest/pmc/metadata/PmcXmlHandlerTest.java

```diff
@@ -261,9 +270,10 @@ public void testElementCitation() throws Exception {
 			ExtractedDocumentMetadata.Builder metaBuilder = ExtractedDocumentMetadata.newBuilder();
 			metaBuilder.setId("some-id");
 			PmcXmlHandler pmcXmlHandler = new PmcXmlHandler(metaBuilder);
-			saxParser.parse(inputStream = PmcXmlHandler.class.getResourceAsStream(filePath), 
-					pmcXmlHandler);
-			ExtractedDocumentMetadata meta = metaBuilder.build();
+			fileReader = new InputStreamReader(PmcXmlHandler.class.getResourceAsStream(filePath), "UTF-8");
+            InputSource inputSource = new InputSource(fileReader);
+            saxParser.parse(inputSource, pmcXmlHandler);
+            ExtractedDocumentMetadata meta = metaBuilder.build();
```

Comment:
Info: i'm not sure if I should change tabs to spaces here, maybe we'll have to talk about it at our meeting with mkobos and mhorst.